doc/_templates/autoapi/python/module.rst

```diff
@@ -6,9 +6,6 @@
 :orphan:
 
 {% endif %}
-:mod:`{{ obj.name }}`
-======={{ "=" * obj.name|length }}
-
```

Comment:
I think this removes the item from the TOC, right? Is that intentional?

Reply:
This does not remove the module from TOC, rather just the title. It will also display more meaningful titles which we have in the docstrings instead of just the module name